terraform/alb.tf

```diff
@@ -264,3 +264,35 @@ resource "aws_alb_listener_rule" "compiler-explorer-alb-listen-https-stats" {
   }
   listener_arn = aws_alb_listener.compiler-explorer-alb-listen-https.arn
 }
+
+resource "aws_alb_listener_rule" "compiler-explorer-alb-listen-http-aarch64prod" {
```

Comment:
I need to see if the http ones are needed any more (or the https ones), basically if the ALB is hitting the HTTPS all the time only then we could kill off half these.